packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -40,6 +40,7 @@ describeCompat("GC Data Store Aliased Full Compat", "FullCompat", (getTestObject
 		return (dataStore._context.containerRuntime as ContainerRuntime).summarize({
 			runGC: true,
 			trackState: false,
```

Comment:
Remove trackState?

Reply:
I will do that in the other PR which removes trackState.